### PR TITLE
[Sprint Maintain] Add SwipeFresh List Manager Leave Request

### PR DESCRIPTION
**Pull request by Tuanlvt**
Add SwipeFresh List Manager Leave Request
Link redmine: https://dev.framgia.com/redmine/issues/130524

### CODE REVIEW

File: app/src/main/java/com/framgia/wsm/utils/binding/BindingUtils.java [hunk 1]
```diff
@@ -152,6 +153,11 @@ public static void loadImage(ImageView imageView, String url) {
         Glide.with(imageView.getContext()).load(uri).dontAnimate().into(imageView);
     }
 
+    @BindingAdapter("colorRes" )
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/utils/binding/BindingUtils.java]: ERROR: ')' is preceded with whitespace.

File: app/src/main/java/com/framgia/wsm/utils/binding/BindingUtils.java [hunk 2]
```diff
@@ -152,6 +153,11 @@ public static void loadImage(ImageView imageView, String url) {
...
+    public static void setRefreshLayout(final SwipeRefreshLayout swipeRefreshLayout, int[] colorResIds) {
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/utils/binding/BindingUtils.java]: ERROR: Line is longer than 100 characters (found 105).

File: app/src/main/res/layout/fragment_manage_list_requests.xml [hunk 3]
```diff
@@ -14,228 +15,234 @@
...
-                                    android:layout_marginLeft="@dimen/dp_5"
-                                    android:layout_marginStart="@dimen/dp_5"
-                                    android:gravity="center_vertical"
-                                    android:text="@{viewModel.fromTime}"
-                                    android:textColor="@color/cardview_dark_background"
-                                    android:textSize="@dimen/sp_17"
+                                    android:orientation="horizontal"
+                                    android:padding="@dimen/dp_7"
+                                    >
+                                    <TextView
```
Comment (trainer-android-fr) [inline app/src/main/res/layout/fragment_manage_list_requests.xml]: WARNING: [TooDeepLayout] `fragment_manage_list_requests.xml` has more than 10 levels, bad for performance
 Layouts with too much nesting is bad for performance. Consider using a flatter layout (such as `RelativeLayout` or `GridLayout`).The default maximum depth is 10 but can be configured with the environment variable `ANDROID_LINT_MAX_DEPTH`.

File: app/src/main/java/com/framgia/wsm/screen/managelistrequests/ManageListRequestsViewModel.java [hunk 4]
```diff
@@ -61,6 +65,13 @@
     private ActionRequest mActionRequest;
     private int mItemPosition;
     private int mAction;
+    private boolean mIsSwipeFresh;
+    private final int[] mSwipeColors = {
+            Color.parseColor(ColorManagers.COLOR_GREEN_DARK),
+            Color.parseColor(ColorManagers.COLOR_STATUS_RED),
+            Color.parseColor(ColorManagers.COLOR_STATUS_BLUE_DARK)
+    };
+    private boolean isRefreshEnable;
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/screen/managelistrequests/ManageListRequestsViewModel.java]: WARNING: [AvoidFieldNameMatchingMethodName] Field isRefreshEnable has the same name as a method
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#AvoidFieldNameMatchingMethodName

File: app/src/main/java/com/framgia/wsm/screen/managelistrequests/ManageListRequestsViewModel.java [hunk 5]
```diff
@@ -61,6 +63,8 @@
     private ActionRequest mActionRequest;
     private int mItemPosition;
     private int mAction;
+    private boolean mIsSwipeFresh;
+    private boolean isRefreshEnable;
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/screen/managelistrequests/ManageListRequestsViewModel.java]: WARNING: [AvoidFieldNameMatchingMethodName] Field isRefreshEnable has the same name as a method
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#AvoidFieldNameMatchingMethodName

File: app/src/main/java/com/framgia/wsm/screen/managelistrequests/ManageListRequestsViewModel.java [hunk 6]
```diff
@@ -61,6 +63,8 @@
     private ActionRequest mActionRequest;
     private int mItemPosition;
     private int mAction;
+    private boolean mIsSwipeFresh;
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/screen/managelistrequests/ManageListRequestsViewModel.java]: pls rename to ``isLoading``

File: app/src/main/java/com/framgia/wsm/screen/managelistrequests/ManageListRequestsViewModel.java [hunk 7]
```diff
@@ -61,6 +63,8 @@
     private ActionRequest mActionRequest;
     private int mItemPosition;
     private int mAction;
+    private boolean mIsLoading;
+    private boolean isRefreshEnable;
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/screen/managelistrequests/ManageListRequestsViewModel.java]: WARNING: [AvoidFieldNameMatchingMethodName] Field isRefreshEnable has the same name as a method
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#AvoidFieldNameMatchingMethodName

**Pull request merged**